### ISSUE TITLE
Feature Request: Add Support for Musical Interludes Between Chapters

### ISSUE THREAD

**Issue by auzziejay** [DrewThomasson/ebook2audiobook#155]
It would be great to have the ability to include short sounds or musical interludes between chapters in the generated audiobooks. This feature could enhance the listening experience by providing smooth transitions and a polished, professional touch to the output.

**Comment by ROBERT-MCDOWELL** [comment]
yes, sure... we can even ad an A..I. making a background music for the entire book if you wish.... but this is another extra feature and complex thing. each feature must work for all language and it's a big challenge. let's make the essential things work and be stable ok?
